### DISCUSSION TITLE
System.Net.WebException: Error: NameResolutionFailure

### DISCUSSION THREAD

**Discussion by realwld** [naverz/zepeto-studio-kor#381] [Issue Report (Legacy)]
![image](https://user-images.githubusercontent.com/65853706/153843711-eda91923-db54-4479-b239-20d4287f97bf.png)

SampleScene을 실행하니 다음과 같은 에러가 뜨는데, 혹이 이 에러는 무엇때문에 일어나는 것인가요?

유니티 버전은 2020.3.9.f1 Personal 버전
제페토 월드 버전은 1.1.4 입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 일시적으로 네트워크 응답이 없었던 것 같습니다. 
월드 최신 버전 1.1.5으로 업데이트를 하셨는지요?
여전히 관련 오류가 발생한다면 말씀 주세요~
